pom.xml

```diff
@@ -48,24 +48,21 @@
         <air.check.skip-pmd>true</air.check.skip-pmd>
 
         <!-- overriding version from airbase -->
```

Comment:
If there are no overrides left.. remove this comment

Reply:
done.